libmamba/src/core/util.cpp

```diff
@@ -301,6 +301,33 @@ namespace mamba
                 line.pop_back();
             }
 
+            const auto lstrip_line = util::lstrip(line);
```

Comment:
@Klaim: would you prefer this
```suggestion
            line = util::lstrip(line);
```

with subsequent modifications?

Reply:
Yes that 's what I meant (the move is unnecessary).

Reply:
Done in 63348f9.